cornflow/tests/custom_test_case.py

```diff
@@ -2,6 +2,7 @@
 This file contains the different custom test classes used to generalize the unit testing of cornflow.
 """
 # Import from libraries
+from abc import ABC
```

Comment:
are you using this? I do not see any reference to it.

Reply:
No, this was a mistake